benchmarks/WebJobs.Script.Benchmarks/SerializationBenchmarks.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
You can back this out... we don't need it checked in.

Reply:
Removed!